internal/loader/aftermarket_device_loader.go

```diff
@@ -56,11 +57,18 @@ func (ad *AftermarketDeviceLoader) BatchGetLinkedAftermarketDeviceByVehicleID(ct
 
 	for i, vID := range vehicleIDs {
 		if am, ok := amByVehicleID[vID]; ok {
-			imageUrl := helpers.GetAftermarketDeviceImageUrl(ad.settings.BaseImageURL, am.ID)
+			var retErr error
+			imageUrl, err := aftermarket.GetAftermarketDeviceImageUrl(ad.settings.BaseImageURL, am.ID)
```

Comment:
```suggestion
			imageURL, err := aftermarket.GetAftermarketDeviceImageUrl(ad.settings.BaseImageURL, am.ID)
```

Similarly in other places.

It's weird that this call can fail. I guess the culprit is [`url.Parse`](https://cs.opensource.google/go/go/+/refs/tags/go1.22.2:src/net/url/url.go;l=1266).

Reply:
Is it resolved? I think Go casing is `imageURL`.

Reply:
I made a PR into your PR #94